orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/model/PipelineTemplate.java

```diff
@@ -37,6 +37,7 @@
   private List<StageDefinition> stages;
   private List<TemplateModule> modules;
   private List<PartialDefinition> partials = new ArrayList<>();
+  private PipelineId pipelineId;
```

Comment:
I seem to be losing the `pipeline` after the merge of templates and after `pipelineId` tag has resolbed. I think I should carry the value from one of the transform? @robzienert